data/styles/clear/include/Icons.mapcss

```diff
@@ -2526,27 +2530,37 @@ area|z16-[amenity=parking][parking=lane],
 /* =no doesn't work in kothic */
 /* node|z16-[amenity=parking][access=no], */
 /* area|z16-[amenity=parking][access=no], */
-{icon-image: ;icon-min-distance: 0;}
+node|z16-[amenity=parking][parking=street_side][fee?],
+area|z16-[amenity=parking][parking=street_side][fee?],
+node|z16-[amenity=parking][parking=lane][fee?],
+area|z16-[amenity=parking][parking=lane][fee?],
+{icon-image: none;icon-min-distance: 0;}
```

Comment:
this icons reset block should go after the paid parking icon block below, as the letter will put icons for all zooms again :)

also you've added small `parking_pay-s.svg` icons in the vehicle style, but forgot here?

Reply:
thank you, fixed icon block :) 
the parking_pay-s.svg is two blocks below